website/src/releases/2023-03/deployment-previews.mdx

```diff
@@ -0,0 +1,15 @@
+---
+date: 2023-03-03
+title: Custom Applications deployment previews
+description: Introducing deployment previews for Custom Applications.
+type: announcement
+topics:
+  - Merchant Center
+  - Configuration
+---
+
+Custom Applications now let you generate [deployment previews](/concepts/deployment-previews), where you can test new features without affecting the production version of your application.
+
+If you're using the [legacy workaround](/help-needed#legacy-workaround) by creating multiple Custom Applications with different entry points, we recommend migrating to deployment previews for a better developer experience with less maintenance, where you only need to configure one Custom Application.
```

Comment:
```suggestion
If you're using the [legacy workaround](/help-needed#legacy-workaround) of creating multiple Custom Applications with different entry points, we recommend migrating to deployment previews. They provide a better developer experience with less maintenance since you only need to configure one Custom Application.
```

Reply:
@Anshuman71 What about this suggestion? ⬆️